config.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+	"github.com/BurntSushi/toml"
+)
+
+func parseConfig() tomlConfig {
```

Comment:
Rename this to `parseConfigFile` and give it a function comment (even though it's not strictly necessary)